substratest/assets.py

```diff
@@ -1,10 +1,11 @@
 import abc
 import enum
-import dataclasses
 import re
 import time
 import typing
+import pydantic
 
+from inspect import isclass
```

Comment:
To follow PEP8, you must group the import into 3 groups (https://www.python.org/dev/peps/pep-0008/#imports):
- standard library
- 3rd party library
- internal library

Reply:
Yes, I was actually checking the doc to be sure that I made it right, I'll change that